### DISCUSSION TITLE
4.10.0 升级启动DbType为空

### DISCUSSION THREAD

**Discussion by teng-jy** [zuihou/zuihou-admin-cloud#171] [02.提问]
![d66cfd62f4e947373958440a5f28025](https://user-images.githubusercontent.com/62694039/185562078-c6398433-3319-4649-ba59-df31c004d309.jpg)

其中005.lamp-boot-pro-datasource-column启动.md文档中第4条没看懂 感觉这里出的问题

**Comment by zuihou** [comment]
更新代码
